crates/cli/commands/src/launcher.rs

```diff
@@ -0,0 +1,52 @@
+use async_trait::async_trait;
+use futures::Future;
+use reth_cli::chainspec::ChainSpecParser;
+use reth_db::DatabaseEnv;
+use reth_node_builder::{NodeBuilder, WithLaunchContext};
+use std::{fmt, marker::PhantomData, sync::Arc};
+
+#[async_trait(?Send)]
```

Comment:
do we even need async_trait here anymore? 
could maybe even use `fn entrypoint<'a> -> impl Future`?

Reply:
Implemented using `impl Future`,`async_trait` dependency removed.